README.md

```diff
@@ -226,13 +233,21 @@ Returns a `Promise` which resolves to an array of search result `Ad` objects.
 const kijiji = require("kijiji-scraper");
 
 let options = {
-    minResults: 40
+    minResults: 40 // Use only multiples of 20 as Kijiji renders pages in units of 20 listings per page
 };
 
 let params = {
-    locationId: 1700185,  // Same as kijiji.locations.ONTARIO.OTTAWA_GATINEAU_AREA.OTTAWA
-    categoryId: 27,  // Same as kijiji.categories.CARS_AND_VEHICLES
-    sortByName: "priceAsc"  // Show the cheapest listings first
+    locationId: 1700185,  // REQUIRED. Same as kijiji.locations.ONTARIO.OTTAWA_GATINEAU_AREA.OTTAWA
```

Comment:
Minor: could you format the spacing so all of these comments line up?

Also could you add a comment to indicate that these are not necessarily the _only_ parameters